contrib/ec2/deis.template.json

```diff
@@ -298,6 +298,12 @@
             "Protocol": "HTTP"
           },
           {
+            "InstancePort": "443",
+            "InstanceProtocol": "HTTPS",
+            "LoadBalancerPort": "443",
+            "Protocol": "HTTPS"
+          },
```

Comment:
Good catch. I was thinking in the context of installing ssl on the routers via https://github.com/deis/deis/pull/2194 which listens on port 443 but in EC2's case it'd be better to install the certs on the load balancer instead.